manifests/p/ppy/osu/2024.412.1/ppy.osu.locale.en-US.yaml

```diff
@@ -0,0 +1,37 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: ppy.osu
+PackageVersion: 2024.412.1
+PackageLocale: en-US
+Publisher: ppy Pty Ltd
+PublisherUrl: https://github.com/ppy
+PublisherSupportUrl: https://github.com/ppy/osu/issues
+PackageName: osu!
+PackageUrl: https://github.com/ppy/osu
+License: MIT
+LicenseUrl: https://github.com/ppy/osu/raw/master/LICENCE
+Copyright: Copyright (c) 2024 ppy Pty Ltd
+CopyrightUrl: https://github.com/ppy/osu/raw/master/LICENCE
+ShortDescription: rhythm is just a *click* away!
+Description: |-
+  A free-to-win rhythm game. Rhythm is just a click away!
+
+  This is the future – and final – iteration of the osu! game client which marks the beginning of an open era! Currently
+  known by and released under the release codename "lazer". As in sharper than cutting-edge.
+Tags:
+- beatmap-parser
+- game
+- hacktoberfest
+- osu
+- rhythm
+ReleaseNotesUrl: https://github.com/ppy/osu/releases/tag/2024.412.1
+InstallationNotes: |-
+  This is the future – and final – iteration of the osu! game client which marks the beginning of an open era! Currently
+  known by and released under the release codename "lazer". As in sharper than cutting-edge.
+Documentations:
+  - DocumentLabel: Wiki
+    DocumentUrl: https://github.com/ppy/osu/wiki
+PurchaseUrl: https://osu.ppy.sh/home/support
```

Comment:
Not sure this makes sense. The game doesn't need to be purchased. Donations are completely optional and I'd rather not have this listed on a package.